seisbench/util/torch_helpers.py

```diff
@@ -18,3 +18,26 @@ def worker_seeding(wid):
     """
     uint64_seed = torch.initial_seed()
     np.random.seed([uint64_seed >> 32, uint64_seed & 0xFFFF_FFFF])
+
+
+def min_max_normalization(x: np.array, eps: float = 1e-10) -> np.array:
+    """
+    Min-max normalize a numpy array, i.e. values are in the range [0, 1].
+
+    :param x: numpy array of arbitrary shape
+    :param eps: Float to avoid division by zeros
+    :return: min-max normalized input array
+    """
+    return (x - np.min(x)) / (np.max(x) - np.min(x) + eps)
+
+
+def z_score_normalization(x: np.array) -> np.array:
+    """
+    Normalize data by z-score
+    .. math::
```

Comment:
The math doesn't render correctly. I think it needs an additional blank line before the `.. math::` statement.